Sources/PointFree/PublicEpisodes/0000-Introduction.swift

```diff
@@ -2,12 +2,12 @@ import Foundation
 
 let introduction = Episode(
   blurb: """
-Point-Free is here, bringing you videos covering functional programming concepts using the Swift language.
+Point-Free is here, bringing you videos covering functional programming concepts using the Swift language. \
```

Comment:
best to not have newlines for episode blurbs cause some readers will render them